src/generators/Sequence.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Created by chetanv on 01/06/16.
+ */
+
+import Generator from './Generator';
+
+class Sequence extends Generator {
+  static sequences = {};
+  id = '';
+
+  constructor(factoryGirl, id, callback = null) {
```

Comment:
Just to add, the same problem would exist with current version as well :(. Seems like people haven't hit it yet to raise an issue, or probably using some workaround.